python/cuml/multiclass/multiclass.py

```diff
@@ -0,0 +1,225 @@
+        return self.multiclass_estimator.n_classes_
+
+    @generate_docstring(y='dense_anydtype')
+    def fit(self, X, y) -> 'MulticlassClassifier':
+        """ Fit a multiclass classifier. """
+        if self.strategy == 'ovr':
+            self.multiclass_estimator = sklearn.multiclass.\
+                OneVsRestClassifier(self.estimator, n_jobs=None)
+        elif self.strategy == 'ovo':
+            self.multiclass_estimator = \
+                sklearn.multiclass.OneVsOneClassifier(
+                    self.estimator, n_jobs=None)
+        else:
+            raise ValueError('Invalid multiclass strategy ' +
+                             str(self.strategy) + ', must be one of '
+                             '{"ovr", "ovo"}')
+        X, _, _, _, _ = input_to_host_array(X)
+        y, _, _, _, _ = input_to_host_array(y)
+        with cuml.internals.exit_internal_api():
+            return self.multiclass_estimator.fit(X, y)
+
+    @generate_docstring(return_values={'name': 'preds',
+                                       'type': 'dense',
+                                       'description': 'Predicted values',
+                                       'shape': '(n_samples, 1)'})
+    def predict(self, X) -> CumlArray:
+        """ Predict using multi class classifier. """
+        X, _, _, _, _ = input_to_host_array(X)
+        with cuml.internals.exit_internal_api():
+            preds = self.multiclass_estimator.predict(X)
+        return preds
+
+    @generate_docstring(return_values={'name': 'results',
+                                       'type': 'dense',
+                                       'description': 'Decision function \
+                                       values',
+                                       'shape': '(n_samples, 1)'})
+    def decision_function(self, X) -> CumlArray:
+        """ Calculate the decision function. """
+        X, _, _, _, _ = input_to_host_array(X)
+        with cuml.internals.exit_internal_api():
+            df = self.multiclass_estimator.decision_function(X)
+        return df
+
+    def get_param_names(self):
+        return super().get_param_names() + ['estimator', 'strategy']
+
+
+class OneVsRestClassifier(MulticlassClassifier):
+    """ Wrapper around Sckit-learn's class with the same name. The input can be
+    any kind of cuML compatible array, and the output type follows cuML's
+    output type configuration rules.
+
+    Berofe passing the data to scikit-learn, it is converted to host (numpy)
+    array. Under the hood the data is partitioned for binary classification,
+    and it is transformed back to the device by the cuML estimator. These
+    copies back and forth the device and the host have some overhead. For more
+    details see issue https://github.com/rapidsai/cuml/issues/2876.
+
+    For documentation see `scikit-learn's OneVsOneClassifier
+    <https://scikit-learn.org/stable/modules/generated/sklearn.multiclass.OneVsRestClassifier.html>`_. # noqa: E501
```

Comment:
Did this give you a flake error? Normally it can parse full URLs and not generate the warning. I only mention this because it shows up in the documentation:
![image](https://user-images.githubusercontent.com/42954918/100830330-9d71a280-3420-11eb-90bb-f4ab97cc9199.png)